src/controllers/Collection.js

```diff
@@ -146,6 +146,16 @@ class CollectionController extends BaseController {
     }, options)
       .then(response => response.result);
   }
+
+  delete (index, collection) {
+    const request = {
+      index,
+      collection,
+      action: 'delete'
+    };
+    return this.query(request)
+      .then(undefined);
```

Comment:
Then `then` method take a callback as a parameter, passing `undefined` will result to a `TypeError`
```suggestion
      .then(() => undefined);
```